components/eamxx/src/share/io/scorpio_output.cpp

```diff
@@ -500,6 +500,19 @@ run (const std::string& filename,
     // then there's no point in copying from the field's view to dev_view
     if (not is_aliasing_field_view) {
       switch (rank) {
+        case 0:
+        {
+          auto new_view_0d = field.get_view<const Real,Device>();
+          auto avg_view_0d = view_Nd_dev<0>(data);
+          Kokkos::parallel_for(policy, KOKKOS_LAMBDA(int i) {
```

Comment:
```suggestion
          Kokkos::parallel_for(policy, KOKKOS_LAMBDA(int) {
```
I think this _might_ cause a warning for unused variable since `i` is never used, not completely sure. I would exclude `i` from the lambda signature just to make it clear it is not needed.

Or if that is more confusing, we could cast it to void
```
Kokkos::parallel_for(policy, KOKKOS_LAMBDA(int i) {
  (void)i;
  ...
});
```